.github/actions/goreleaser-build-sign-publish/README.md

```diff
@@ -0,0 +1,125 @@
+# goreleaser-build-sign-publish
+> goreleaser wrapper action
+
+## workflows
+
+### build publish
+
+```yaml
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    environment: release
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.aws-role-arn }}
+          role-duration-seconds: ${{ secrets.aws-role-dur-sec }}
+          aws-region: ${{ secrets.aws-region }}
+      - name: Cache macos sdk
+        id: sdk-cache
+        uses: actions/cache@v3
+        with:
+          path: MacOSX12.3.sdk
```

Comment:
Newest version is 13.x IIRC

Reply:
Do we build MacOS versions as well? I don't see them

Reply:
@samsondav  For develop, we only build the linux binaries (used for the docker images). The `.goreleaser.develop.yaml` is used there. You can build the macos binaries by running goreleaser w/ the default config file locally (`.goreleaser.yaml`). This was done b/c we dont want to push binaries to a github release from develop. What we could do in the future is push binaries created from develop to s3 and pull them with an asdf plugin. For the macos version, the latest available prebuilt is [12.3](https://github.com/joseluisq/macosx-sdks). We implemented this on the bif repo and it seems to work fine.

Reply:
@ajgrande924 we can publish our own prebuilt if we really need 13